pyproject.toml

```diff
@@ -1,17 +1,111 @@
 [build-system]
-requires = ["setuptools >= 30.3.0", "wheel"]
+requires = ["setuptools >= 66", "setuptools-scm >= 7.1"]
+build-backend = "setuptools.build_meta"
 
 
-[tool.black]
-target-version = ["py39", "py310", "py311"]
-line-length = 80
+[project]
+name = "Mopidy-Spotify"
+description = "Mopidy extension for playing music from Spotify"
+readme = "README.rst"
+requires-python = ">= 3.11"
+license = { text = "Apache-2.0" }
+authors = [{ name = "Stein Magnus Jodal", email = "stein.magnus@jodal.no" }]
```

Comment:
Maybe we need a `maintainers` key. But I am not bothered, you can forward me the 2 emails that come through!

Reply:
There is support for it! https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#authors-maintainers